templates/renovate.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "ignoreDeps": [
```

Comment:
@suztomo How did you come up with this configuration?

Reply:
This is copy from main branch.